api/v1alpha1/queue_types.go

```diff
@@ -31,8 +31,8 @@ type ClusterQueueReference string
 
 // QueueStatus defines the observed state of Queue
 type QueueStatus struct {
-	// PendingWorkloads is the number of workloads currently assigned to this
-	// queue not yet assigned to a ClusterQueue.
+	// PendingWorkloads is the number of workloads currently admitted to this
+	// queue not yet admitted to a ClusterQueue.
```

Comment:
in this queue not yet admitted to a ClusterQueue?

Reply:
Will address in a followup PR